automation/roles/patroni/tasks/config.yml

```diff
@@ -1,5 +1,34 @@
 ---
 # Patroni configure
+
+# Ensure archive_command is set correctly for pgbackrest (same logic as pre_checks/pgbackrest.yml)
```

Comment:
I don't quite understand why we should duplicate the logic here, because we are already set archive_command in pre_checks role.